inst/extdata/mass-balance-settings.json

```diff
@@ -0,0 +1,100 @@
+{
+    "MassBalancePlots": [
+        {
+            "Name": "Name of mass balance sub-section (optional)",
+            "Molecules": [
+                "Aciclovir"
+            ],
+            "Groupings": [
+                {
+                    "Name": "Intracellular",
+                    "Include": [
+                        "Organism|**|Intracellular|Aciclovir*"
```

Comment:
I forgot what we decided: if the molecules list above includes only `Aciclovir`, but I have e.g. `Aciclovir-XY-Metabolite` in my simulation: will `Aciclovir-XY-Metabolite` be included via `"Organism|**|Intracellular|Aciclovir*"` or not?

Reply:
Yes, I stated it in the vignette


Note that inclusion and exclusion criteria for molecule paths leverage [wild card symbols (`*`)](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/load-get.html#accessing-entities-of-the-model-and-their-properties---the-path-concept).
...
Another use case is when a metabolite is defined using the molecule name as prefix.
Then, the metabolite can be included in the grouping.

```json
{
  "Name": "Plasma for Aciclovir and metabolites", 
  "Include": ["Organism|**|Plasma|Aciclovir*"]
}
```

Reply:
What I meant: the molecules list in the example contains `Aciclovir `only:
```
            "Molecules": [
                "Aciclovir"
            ],
```
Now if `"Include": ["Organism|**|Plasma|Aciclovir*"]` retrieves both `Aciclovir` and `Aciclovir-XY-Metabolite`: will the metabolite be included or not?

Reply:
Since only aciclovir was in molecules, no, they should not be retreived from the search.
(because the interesection between all the molecule paths from the compound in Molecules field and the Include field is used).

Reply:
ok. it would be good to select another example for vignette, which incorporates metabolites as well - but let's keep Aciclovir for now AS IS.

Reply:
created an issue for that: https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/issues/1148